src/components/Book.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import './Book.css';
+import BookRecord from './BookRecord';
+
+function Book() {
+  const books = [
+    {
+      id: 1,
+      type: 'action',
+      title: 'The Hunger Games',
+      author: 'Suzanne Collins',
+    },
+    {
+      id: 2,
+      type: 'action',
+      title: 'Dune',
+      author: 'Frank Herbert',
+    },
+    {
+      id: 3,
+      type: 'action',
+      title: 'Capital in the Twenty-First Century',
+      author: 'Suzanne Collins',
+    },
+  ];
+  return (
+    <>
+      <section className="section">
+        <div className="container">
+          <div>
+            {books.map((book) => (
+              <BookRecord
+                type={book.type}
+                title={book.title}
+                author={book.author}
+                key={book.id}
+              />
+            ))}
+          </div>
+        </div>
+      </section>
+      <hr />
+      <h2>ADD NEW BOOK</h2>
+      <form action="">
+        <div className="formcontrol">
+          <input
+            type="text"
+            className="input-text"
+            placeholder="Add title"
+            name="title"
+          />
+          <input
+            type="text"
+            className="input-text"
+            placeholder="Add author"
+            name="author"
+          />
+        </div>
+      </form>
```

Comment:
- [ ] as part of the requirements the add new book form should be created in a separate component , kindly fix that 🙏 

![image](https://github.com/abenezer-addisu/React-Bookstore/assets/89282221/f275bb3b-fe5b-4ddd-9715-2c49029d0426)

Reply:
Done